### ISSUE TITLE
Schedule edit link is not working

### ISSUE THREAD

**Issue by euclidian** [kartoza/qgis.org.za#134]
When we click on edit link at schedule page, it doesn't do anything

**Comment by gubuntu** [comment]
I don't think it is supposed to do anything according to the documentation (i.e. it should not even be there) so low priority. 

Once the structure is created with an upload, we should be able to allocate items to slots through admin (you can test that :-))
